src/backend/ai-reviewer-api/jag-reviewer-api.yaml

```diff
@@ -2,14 +2,14 @@ openapi: 3.0.0
 info:
   version: 0.7.0
   title: jag-reviewer-api
-  description: Efiling AI Reviewer api
+  description: AI Reviewer api
```

Comment:
Should be 'Ai Reviewer api' for consistency?

Reply:
Yeah, good.  Done.